Dockerfile

```diff
@@ -13,7 +13,12 @@ RUN cargo install --locked --path .
 
 FROM debian:buster-slim
 
-RUN apt update && apt install -y bash curl netcat tini wget
+RUN apt update && apt install -y ca-certificates
+
+#Let debian repos be fetched securely:
+RUN sed -i "s/http:\/\//https:\/\//g" /etc/apt/sources.list
+
+RUN apt update && apt upgrade -y && apt install -y bash curl netcat tini wget
```

Comment:
Why not do all the `apt installs` in one command?

Reply:
Since ca-certificates isn't there (or isn't up to date), fetching everything immediately with https:// errors because it doesn't trust the cert.  So 1 to install ca-certificates to get the latest trusted root certs, then the 2nd invocation to actually fetch the remaining desired packages over https://